supporting_scripts/hyperion/consistency-check-benchmark/consistency_check.py

```
@@ -84,16 +90,30 @@ def consistency_check(session: requests.Session, exercise_ids: Dict[str, int]) -
+            local_id = future_to_local_id[future]
             try:
                 result = future.result()
-                logging.info(result)
+                if "success" in result:
+                    succeeded_count += 1
+                    logging.info(f"[OK]   {result}")
+                else:
+                    failed_count += 1
+                    failed_checks.append(local_id)
+                    logging.error(f"[FAIL] {result}")
             except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)